src/main/java/Handler/RootHandler.java

```diff
@@ -26,12 +28,20 @@ private Response buildGetResponse() {
     int contentLength = ResponseHeader.determineContentLength(messageBody);
     return new Response.Builder(HttpStatusCode.OK)
                        .setHeader(ResponseHeader.CONTENT_LENGTH, contentLength)
+                       .setHeader(ResponseHeader.CONTENT_TYPE, "text/html")
                        .messageBody(messageBody)
                        .build(); 
   }
 
   private String createMessageBody() {
-    return this.store.listContent();
+    String htmlResponse = "";
+    String[] files = this.store.listContent();
+    for (String file : files) {
+      htmlResponse += String.format(
+        "<a href=\"http://localhost:%d/%s\">%s</a><br>", this.port, file, file);
```

Comment:
If I asked you to deploy your server on some service (Heroku/AWS/etc.), this wouldn't work since you are hardcoding this to `localhost`. We shouldn't be hardcoding the hostname here.